tinygrad/helpers.py

```diff
@@ -15,7 +15,11 @@ def prod(x:Iterable[T]) -> Union[T,int]: return functools.reduce(operator.mul, x
 # NOTE: helpers is not allowed to import from anything else in tinygrad
 OSX = platform.system() == "Darwin"
 CI = os.getenv("CI", "") != ""
-
+def progress_bar(iterable, length=50, fill='█'):
+    start = time.time(); print('{0}%|{1} 0/{2} 00:00<?,?it/s'.format(0, ' ' * length, len(iterable)), end='\r')
```

Comment:
don't use `;` to combine lines, and indent 2

also don't update files in extra (those are not tested, and having tqdm there is fine) and add tests